### DISCUSSION TITLE
q-input event is not triggered when it's inside q-menu

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14748] [General - Components / Directives / etc] [answered]
Dear Quasar team,

Could you please have a look of this issue, thanks!
I reproduce this issue in the link https://codepen.io/Stanley-549393092/pen/QWxjdOo?editors=1011

If input something, the change & blur event can be triggered when the cursor is inside the area of q-menu.
However, when moving the cursor outside of the q-menu (that means I want to close the menu), it can not be triggered.

Regards

**Comment by pdanpdan** [comment, marked as answer]
Yes, the menu gets destroyed and the events are no longer processed
Listen for `@hide` and keep a temp value of the input
https://codepen.io/pdanpdan/pen/gOKaGEW?editors=1010

**Reply by 549393092** [reply]
@pdanpdan Thanks a lot for your help! Your solution inspired me and I implement it in this way.
https://codepen.io/Stanley-549393092/pen/xxzwjMz?editors=1011
